.circleci/config.yml

```diff
@@ -178,6 +186,113 @@ commands:
       - store_artifacts:
           path: ~/scalyr-agent-dev/log
 
+  smoke-standalone-tox:
+    description: "A base command for all tox based smoke test jobs"
+    parameters:
+      python_version:
+        description: "Python version to use (e.g. 3.5, 3.6, 3.7, 3.8, etc.)."
+        type: string
+      tox_version:
+        description: "tox package version to use."
+        type: string
+        default: "3.14.3"
+      apt_dependencies:
+        description: "Any optional apt dependencies which should be installed."
+        type: string
+        default: ""
+      tox_target:
+        description: "Tox target to run."
+        type: string
+      store_coverage:
+        description: "True to persist / store coverage data."
+        type: boolean
+        default: false
+    steps:
+      - checkout
+      - restore_cache:
+          key: deps2-tox-{{ .Branch }}-<< parameters.python_version >>-venv-{{ checksum "dev-requirements.txt" }}
+      - when:
+          condition: << parameters.apt_dependencies >>
+          steps:
+            - run:
+                name: Install APT Dependencies
+                command: |
+                  apt-get update -y
+                  apt-get install -y << parameters.apt_dependencies >>
+      - run:
+          name: Install Dependencies
+          command: |
+            pip install "tox==<< parameters.tox_version >>"
+      - run:
+          name: Run Unit Tests under Python << parameters.python_version >>
+          command: |
+            export AGENT_HOST_NAME=agent-smoke-standalone-${CIRCLE_BUILD_NUM}
+            tox -e<< parameters.tox_target >>
+      - save_cache:
+          key: deps2-tox-{{ .Branch }}-<< parameters.python_version >>-venv-{{ checksum "dev-requirements.txt" }}
+          paths:
+            - "~/.cache/pip."
+
+      - run:
+          name: "Show agent.log"
```

Comment:
Another option would be to persist it as an artifact, similar as we do for benchmarking jobs.

Reply:
I thought that `agent.log`  is not so big, so it would be faster just to look on output, but saving it as an artifact is a better approach. Done.